packages/gamut-styles/src/utilities/boxShadow.ts

```diff
@@ -0,0 +1,37 @@
+import { css } from '@emotion/core';
+
+export type ShadowDepths = 1 | 2 | 3 | 4 | 5;
```

Comment:
The convention here would normally be singular:

```suggestion
export type ShadowDepth = 1 | 2 | 3 | 4 | 5;
```